dashboard/config/locales/slides.en.yml

```diff
@@ -2451,3 +2451,76 @@ en:
+      4:
+        image: 'notes/teaching_cs_fundamentals_4.png'
+        text: "The learning platform will track your progress, so you can pick up where you left off at any time."
+    what_is_cs:
+      1:
+        image: 'notes/what_is_cs.png'
+        text: "Coding is the set of instructions you give to a computer. Computer science is a field where people use the power of computers to solve big problems. Computer science includes coding, but it also includes other things, like how the internet works and impacts society, what to do with big data, and how to break down big problems. Computer science is about problem solving, critical thinking, and persistence."
+    writing_first_program:
+      1:
+        image: 'notes/writing_first_program_1.png'
+        text: "Code.org uses a visual, drag and drop programming language called Blockly. Blockly uses colorful instructions called blocks to build programs and solve puzzles."
+      2:
+        image: 'notes/writing_first_program_2.png'
+        text: "Each puzzle has four parts."
+      3:
+        image: 'notes/writing_first_program_3.png'
+        text: "The play area is on the left, where your program runs."
+      4:
+        image: 'notes/writing_first_program_4.png'
+        text: "The center has the toolbox area, which has all the blocks you need."
+      5:
+        image: 'notes/writing_first_program_5.png'
+        text: "On the right is the workspace, where you will drag blocks to build your program."
+      6:
+        image: 'notes/writing_first_program_6.png'
+        text: "Above the workspace are the instructions for each puzzle."
+      7:
+        image: 'notes/writing_first_program_7.png'
+        text: "If you drag out too many blocks from the toolbox, just drag the extra block back into the toolbox.  Hit run to try your program, and reset to try again."
+    repeat_block:
+      1:
+        image: 'notes/repeat_block.png'
+        text: "A repeat block lets you loop the code inside of it as many times as you tell it to."
+    what_is_an_event:
+      1:
+        image: 'notes/what_is_an_event.png'
+        text: "An event tells your program to listen for something to happen. When that thing happens, the program performs an action. A computer might listen for a mouse click, press on an arrow button, or a tap on the screen."
+    lesson_planning:
+      1:
+        image: 'notes/lesson_planning_1.png'
+        text: "CS Fundamentals lessons begin with overviews to let you know what the lesson will cover."
+      2:
+        image: 'notes/lesson_planning_2.png'
+        text: "Lessons have warm ups, activities, and wrap ups. If you want to create your own lesson plans, you might want to keep these tips in mind."
```

Comment:
nit: I believe "wrap ups", being a noun, should be "wrap-ups"

Reply:
Also I believe "warm ups" should be "warm-ups" or "warmups" for the same reason.